ocpi-toolkit-2.2.1/src/main/kotlin/com/izivia/ocpi/toolkit/modules/credentials/services/CredentialsServerService.kt

```diff
@@ -24,6 +26,8 @@ open class CredentialsServerService(
     ): OcpiResponseBody<Credentials> = OcpiResponseBody.of {
         getCredentials(
             serverToken = token,
+            partnerRepository.getPartnerIdByCredentialsServerToken(token)
+                ?: throw OcpiClientInvalidParametersException("Invalid ServerToken ($token)"),
```

Comment:
tokenA is allowed for credentials module (see `PartnerRepository.checkToken` in `HttpUtils`)

```suggestion
            partnerRepository.getPartnerIdByCredentialsServerToken(token)
                ?: partnerRepository.getPartnerIdByCredentialsTokenA(credentialsTokenA = token)
                ?: throw OcpiClientInvalidParametersException(
                "Invalid token ($token) - should be either a TokenA or a ServerToken",
            ),
```